src/diagnostics/CMakeLists.txt

```diff
@@ -71,7 +71,7 @@ set(MODULE_SRC
 # --- Crashpad ---
 # Not building with MinGW, so turned off for MinGW
 include(GetCompilerInfo)
-set(BUILD_CRASHPAD_CLIENT ON)
+set(BUILD_CRASHPAD_CLIENT OFF)
```

Comment:
Why did you decide to turn off crash dumps?

Reply:
It because I can't build the crashpad-handler from source. It looks like the source is there, but is it missing some CMakeList file or what is missing?

Reply:
In this case, you should disable the use of crashpad only for yourself, and not as a whole in the project. (you should probably add the cmake  option)

Reply:
yes I'll rework it to have an global option to disable crashpad.